sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -228,3 +254,26 @@ def __init__(self, subject, event_type, data, data_version, **kwargs):
         kwargs.setdefault('data_version', data_version)
 
         super(EventGridEvent, self).__init__(**kwargs)
+
+    @classmethod
+    def from_dict(cls, event, **kwargs):
+        # type: (Dict, Any) -> EventGridEvent
+        """
+        Returns the deserialized EventGridEvent object when a dict is provided.
+
+        :param event: The dict representation of the event which needs to be deserialized.
+        :type event: dict
+
+        :rtype: EventGridEvent
```

Comment:
same comment as above regarding to the description of the dict accepted key-value items.